Tools/WebKitTestRunner/cocoa/TestRunnerWKWebView.mm

```diff
@@ -545,6 +546,7 @@ - (void)_webView:(WKWebView *)webView willStartInputSession:(id <_WKFormInputSes
 {
     if (self.willStartInputSessionCallback)
         self.willStartInputSessionCallback();
+    inputSession.accessoryViewShouldNotShow = self.suppressInputAccessoryView;
```

Comment:
We want this to be `true` right? Are we setting that anywhere?

Reply:
Yep! The layout test specifies my new test option:

```
<!DOCTYPE html> <!-- webkit-test-runner [ useFlexibleViewport=true suppressInputAccessoryView=true ] -->
…
```